gender_analysis/analysis/proximity.py

```diff
@@ -14,39 +14,70 @@
 GenderTokenCounters = Dict[str, Counter]
 GenderTokenSequence = Dict[str, Sequence[Tuple[str, int]]]
 GenderTokenResponse = Union[GenderTokenCounters, GenderTokenSequence]
+KeyGenderTokenResponse = Dict[Union[str, int], GenderTokenResponse]
 
 
-def _diff_gender_token_counters(gender_token_counters: GenderTokenCounters,
-                                sort: bool = False,
-                                limit: int = 10,
-                                remove_swords: bool = False) -> GenderTokenResponse:
+def _apply_kwargs(key_gender_token_counters: Dict[Union[str, int], GenderTokenCounters],
```

Comment:
Can we try a more descriptive name here? `_apply_optional_result_filters`, `_apply_kwarg_result_filters`, or something like this? That they are optional kwargs is in the function signature and not something I'd probably put in the function name, but moreover a more descriptive name would be great.

Reply:
👍